audiomentations/augmentations/transforms.py

```diff
@@ -888,6 +888,8 @@ def __init__(
         max_snr_in_db=24,
         min_time_between_sounds=4.0,
         max_time_between_sounds=16.0,
+        noise_power=("relative", "absolute"),
+        absolute_noise_rms_db=-20,
```

Comment:
In real life noises will vary in loudness. It would be nice to have two parameters that specify the min and max loudness of the noises to be mixed in. This way the augmentation transform can vary how loud the noises are, e.g. so it could mix in both soft and loud noises in the same audio clip.

Reply:
I coded it this way because I was in the mindset of adding noises with the same power. But what you said is more flexible indeed, I'll fix it :)